x-pack/plugins/reporting/server/lib/create_worker.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// @ts-ignore
```

Comment:
If you're ignoring due to the import being untyped, this should be:

`// @ts-ignore untyped dependency`

Otherwise, maybe add a comment about why these are ignored?

Reply:
Good to know! This is way more preferable, now it will complain if the module is not found.

I had a bad habit of this. I think I have a followup PR now to fix more of these :D

Reply:
oh I misunderstood: that isn't a syntax that causes just that one type of error to be ignored ts